src/NUnitEngine/nunit.engine/Services/ResultWriters/XmlTransformResultWriter.cs

```diff
@@ -81,18 +81,16 @@ public void CheckWritability(string outputPath)
 
         public void WriteResultFile(XmlNode result, TextWriter writer)
         {
-            using (var xmlWriter = new XmlTextWriter(writer))
+            using (var xmlWriter = XmlWriter.Create(writer, new XmlWriterSettings {Indent = true, ConformanceLevel = ConformanceLevel.Auto}))
             {
-                xmlWriter.Formatting = Formatting.Indented;
                 _transform.Transform(result, xmlWriter);
             }
         }
 
         public void WriteResultFile(XmlNode result, string outputPath)
         {
-            using (var xmlWriter = new XmlTextWriter(outputPath, Encoding.Default))
+            using (var xmlWriter = XmlWriter.Create(new StreamWriter(outputPath), new XmlWriterSettings {Indent = true, ConformanceLevel = ConformanceLevel.Auto}))
```

Comment:
If you don't manually do `new StreamWriter`, it allows the XmlWriter to use XmlWriterSettings.CreateWriter. Does this seem like a good thing to do?

```suggestion
            using (var xmlWriter = XmlWriter.Create(outputPath, new XmlWriterSettings { Indent = true, ConformanceLevel = ConformanceLevel.Auto }))
```